ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,46 @@
+Before filling this issue, please check if the bug or feature request already
+exists here <https://github.com/evil-mad/robopaint/issues>
+
+### Description
+
+For a bug: Describe the bug and list the steps you used when the issue occurred.
+include any files necessary to reproduce this by zipping them and attaching them
+to this issue.
+
+#### Steps to reproduce
+  1.Click on this
+  2.Do that
+
+For an enhancement or new feature: Describe your needs.
+
+### Versions
+This is only needed for a bug
+
+#### What operating system are you running?
+Windows XP, Vista, 7, 8, 10
+OS X 10.?
+Linux what distribution?
+
+#### What version of RoboPaint are you running?
+  1.Go to the home screen of RoboPaint, this is the screen you see when you start
+  RoboPaint and can be accessed by clicking on the home icon in the top left
+  corner of the window.
+  2.Below RoboPaint text in the top middle of the home screen you should see text
+  in parenthesizes starting with `v` copy this and put it below.
+
+RoboPaint version:
+
+#### How did you install RoboPaint?
+I did not, running from source
```

Comment:
I considered making these check boxes, but decided against it, as GitHub interprets it as a task list.